### PR TITLE
Fixed spec_helper and added rules in rubocop

### PR DESCRIPTION
**Pull request by desislava453**

### CODE REVIEW

**Review by AleksandrosV — APPROVED** [review]
It works.

**Review by IvayloHris — APPROVED** [review]
Looks nice and works well :)

**Review by gogstar1899 — APPROVED** [review]
Good job.

**Review by okLetsCode — APPROVED** [review]
Looks good!

File: spec/spec_helper.rb [hunk 1]
```diff
@@ -1,17 +1,16 @@
...
-	config.after(:each) do
-		@driver.quit
-	end
-
-end
\ No newline at end of file
+  config.before(:each) do
+    @driver = Selenium::WebDriver.for :chrome
+    @driver.manage.window.maximize
+    @driver.get('https://www.abv.bg/')
```
Comment (anzayats) [inline spec/spec_helper.rb:11]: it is nice practice to store urls in a separate file with other similar variables that are used in the project

**Pull request merged**